📝(content) Release notes

jkcinouye opened the pull request:
On branch content/chromeos-release-notes
Changes to be committed:
new file:   site/en/posts/chromeos-112-113-release-notes.md
new file:   site/en/posts/chromeos-114-release-notes.md
new file:   site/en/posts/chromeos-115-release-notes.md
new file:   site/en/posts/chromeos-116-release-notes.md

**Please describe what your Pull Request does**

A clear and concise description of what your Pull Request does.

**Please tag each issue this Pull Request resolves**

- Resolves #

jkcinouye committed:
Release note content.

On branch content/chromeos-release-notes
Changes to be committed:
new file:   site/en/posts/chromeos-112-113-release-notes.md
new file:   site/en/posts/chromeos-114-release-notes.md
new file:   site/en/posts/chromeos-115-release-notes.md
new file:   site/en/posts/chromeos-116-release-notes.md

jkcinouye committed:
Merge branch 'main' into content/chromeos-release-notes

Snugug commented on site/en/posts/chromeos-112-113-release-notes.md:
```diff
@@ -0,0 +1,33 @@
+
+## ChromeOS 112: Rupee symbol and Screen Capture updates
+
+ChromeOS 112 released on April 6, 2023, introducing several user productivity features and application updates. Two highlights:
```
Instead of highlights for 112 and 113, can we style these like the other items, where we call out what was released directly?

Snugug commented on site/en/posts/chromeos-115-release-notes.md:
```diff
@@ -0,0 +1,41 @@
+Passpoint streamlines Wi-Fi access and eliminates the need for users to find and authenticate a network each time they visit. Once a user accesses the Wi-Fi network offered at a location, the Passpoint-enabled client device will automatically connect upon subsequent visits. Wi-Fi Passpoint is now supported on ChromeOS through supported Android applications. Wi-Fi Passpoint is a set of Wi-Fi mecha ...
+Other ChromeOS 115 updates include:
+
+- **Expanded language packs:** Some Google Text-to-Speech voices that were previously preinstalled are now downloaded over the network when they are needed. This frees up some space on the ChromeOS device.
```
I think this is missing the "Also released in 115" header

Snugug commented on site/en/posts/chromeos-112-113-release-notes.md:
```diff
@@ -0,0 +1,33 @@
+---
+title: 'ChromeOS 112 and 113 improve Screen Capture, screensavers, and security'
```
All of the other titles are "ChromeOS X release notes"; can we normalize the title?

Snugug commented on site/en/posts/chromeos-112-113-release-notes.md:
```diff
@@ -0,0 +1,33 @@
+---
+title: 'ChromeOS 112 and 113 improve Screen Capture, screensavers, and security'
+metadesc: 'The latest ChromeOS release includes updates to Screen Capture, screensavers, and security.'
```
Can we also normalize the descriptions? Some of these are directly descriptive, like 116, where some mention the current version (here, 114)

Snugug reviewed (changes requested):
A couple of small updates, but otherwise looks gtg. thank you!

jkcinouye committed:
Changes to be committed:
	modified:   site/en/posts/chromeos-112-113-release-notes.md
	modified:   site/en/posts/chromeos-115-release-notes.md
	modified:   site/en/posts/chromeos-116-release-notes.md

Updated titles and descriptions for release notes 112, 113, 115, and 116.
